releasenotes/notes/0.24/fix-9798-30c0eb0e5181b691.yaml

```diff
@@ -1,8 +1,8 @@
 ---
 fixes:
   - |
-    The return type of `qiskit.transpiler.passmanager.PassManager.run` is now
+    The return type of :meth:`~.PassManager.run` will now
     always the same as that of its first argument. Passing a single circuit
     returns a single circuit, passing a list of circuits, even of length 1,
     returns a list of circuits.
-    See `#9798 <https://github.com/Qiskit/qiskit-terra/issues/9798>`.
+    See `#9798 <https://github.com/Qiskit/qiskit-terra/issues/9798>`__.
```

Comment:
Just checking that the trailing space here is intentional. Must be for some formatting reason.

Reply:
It's actually an underscore it's needed syntax for the link to work as expected.